### PR TITLE
docs: Add DB name to backup-and-restore

### PR DESCRIPTION
**Pull request by Mraedis**
Add database name to the restore command and document it in the notes
Minor alignment changes

### CODE REVIEW

**Review by bo0tzz — APPROVED** [review]

**Review by danieldietzler — APPROVED** [review]

**Comment by mmomjian** [comment]
I don’t think this is correct/needed. When restoring a pg_dumpall, the Immich database may not yet exist. It’s better practice to connect to the default Postgres database since that will always exist.

**Comment by Mraedis** [comment]
So what happens when you restore a backup that had a different DB name from default then? Is the DB created when running the command? (in which case the flag is unneeded in both cases)

**Comment by mmomjian** [comment]
> So what happens when you restore a backup that had a different DB name from default then? Is the DB created when running the command? (in which case the flag is unneeded in both cases)

Yes, when restoring pg_dumpall it will create all the databases that were present in the old cluster.

**Comment by Mraedis** [comment]
I'll just re-commit with the indent changes and a note on the username.🙂

File: docs/docs/administration/backup-and-restore.md [hunk 1]
```diff
@@ -103,6 +105,7 @@ services:
 Then you can restore with the same command but pointed at the latest dump.
 
 ```bash title='Automated Restore'
+# Be sure to check the username and database name if you changed them from defaults
```
Comment (mmomjian) [inline docs/docs/administration/backup-and-restore.md:108]:
```suggestion
# Be sure to check the username if you changed it from default
```

**Review by mmomjian — APPROVED** [review]

**Pull request merged**